<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Kistl.DalProvider.EF.Tests.nunit" total="74" failures="9" not-run="0" date="2009-02-18" time="10:58:28">
  <environment nunit-version="2.4.7.0" clr-version="2.0.50727.3074" os-version="Microsoft Windows NT 6.0.6001 Service Pack 1" platform="Win32NT" cwd="P:\Kistl\Tests\Kistl.DalProvider.EF.Tests" machine-name="KISTL" user="david" user-domain="kistl" />
  <culture-info current-culture="de-AT" current-uiculture="de-DE" />
  <test-suite name="P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Kistl.DalProvider.EF.Tests.nunit" success="False" time="14.434" asserts="0">
    <results>
      <test-suite name="P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\bin\Debug\Kistl.DalProvider.EF.Tests.exe" success="False" time="14.424" asserts="0">
        <results>
          <test-suite name="Kistl" success="False" time="14.416" asserts="0">
            <results>
              <test-suite name="DalProvider" success="False" time="14.408" asserts="0">
                <results>
                  <test-suite name="EF" success="False" time="14.400" asserts="0">
                    <results>
                      <test-suite name="Tests" success="False" time="14.391" asserts="0">
                        <results>
                          <test-suite name="BaseServerDataObjectTests" success="False" time="6.108" asserts="0">
                            <results>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.AttachToContext" executed="True" success="True" time="1.019" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.AttachToContext_Other_fails" executed="True" success="True" time="0.006" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.DetachFromContext" executed="True" success="True" time="0.214" asserts="4" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.DetachFromContext_Other" executed="True" success="True" time="0.006" asserts="4" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.fails_on_serializing_from_null_stream" executed="True" success="True" time="0.002" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.fails_on_serializing_to_null_stream" executed="True" success="True" time="0.001" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.FromStream_Attached_fails" executed="True" success="True" time="0.040" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.FromStream_creates_correct_Object" executed="True" success="False" time="0.003" asserts="0">
                                <failure>
                                  <message><![CDATA[System.NotImplementedException : Die Methode oder der Vorgang sind nicht implementiert.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.FromStream_creates_correct_Object() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\BaseServerDataObjectTests.cs:Zeile 179.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.FromStream_Null_StreamReader_fails" executed="True" success="True" time="0.002" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.FromStream_WrongType_fails" executed="True" success="True" time="0.015" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.NotifyChange" executed="True" success="True" time="0.001" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.NotifyPropertyChanged_ing" executed="True" success="True" time="0.000" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ObjectState_CreatedObject_Modified" executed="True" success="True" time="0.000" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ObjectState_ObjectWithID_Modified" executed="True" success="True" time="0.000" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ObjectState_should_be_new_after_init" executed="True" success="True" time="0.001" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ObjectState_should_be_Unmodified_after_setting_ID" executed="True" success="True" time="0.000" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.should_be_attached_after_attaching" executed="True" success="False" time="0.001" asserts="0">
                                <failure>
                                  <message><![CDATA[Cannot get context without Kistl.API.{Client,Server} reference]]></message>
                                  <stack-trace><![CDATA[bei Kistl.API.Tests.Skeletons.IPersistenceObjectTests`1.should_be_attached_after_attaching() in P:\Kistl\Tests\Kistl.API.Tests\Tests.Skeletons\IPersistenceObjectTests.cs:Zeile 27.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.should_have_events_attached_after_init" executed="True" success="True" time="0.001" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.should_not_be_attached_when_created" executed="True" success="True" time="0.001" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.should_roundtrip_IPersistenceObject_correctly" executed="True" success="True" time="0.019" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.should_roundtrip_without_errors" executed="True" success="True" time="0.015" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ToFromStream_roundtrips_correctly" executed="True" success="False" time="4.703" asserts="14">
                                <failure>
                                  <message><![CDATA[  Expected: <Kistl.App.Base.Module__Implementation__>
  But was:  null
]]></message>
                                  <stack-trace><![CDATA[bei Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ToFromStream_roundtrips_correctly() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\BaseServerDataObjectTests.cs:Zeile 143.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ToStream_creates_correct_Stream" executed="True" success="False" time="0.028" asserts="1">
                                <failure>
                                  <message><![CDATA[System.NotImplementedException : Die Methode oder der Vorgang sind nicht implementiert.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ToStream_creates_correct_Stream() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\BaseServerDataObjectTests.cs:Zeile 167.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerDataObjectTests.ToStream_Null" executed="True" success="True" time="0.001" asserts="0" />
                            </results>
                          </test-suite>
                          <test-suite name="BaseServerStructObjectTests" success="True" time="0.016" asserts="0">
                            <results>
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.Clone_creates_memberwise_equal_object" executed="True" success="True" time="0.002" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.fails_on_serializing_from_null_stream" executed="True" success="True" time="0.001" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.fails_on_serializing_to_null_stream" executed="True" success="True" time="0.001" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.NotifyPropertyChanged_ing" executed="True" success="True" time="0.002" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.should_roudtrip_members_correctly" executed="True" success="True" time="0.004" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.BaseServerStructObjectTests.should_roundtrip_without_errors" executed="True" success="True" time="0.004" asserts="1" />
                            </results>
                          </test-suite>
                          <test-suite name="KistlDataContextTests" success="False" time="3.960" asserts="0">
                            <results>
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Attach_IDataObject_Existing" executed="True" success="True" time="2.346" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Attach_IDataObject_Existing_Twice" executed="True" success="True" time="0.009" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Attach_IDataObject_Existing_Twice_But_Different" executed="True" success="True" time="0.019" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Attach_IDataObject_New" executed="True" success="True" time="0.010" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.AttachedObjects" executed="True" success="True" time="0.010" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.CleanUpSession" executed="True" success="True" time="0.008" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.ContainsObject" executed="True" success="True" time="0.011" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.ContainsObject_Not" executed="True" success="True" time="0.009" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Create_Generic" executed="True" success="True" time="0.012" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Create_ObjectType" executed="True" success="True" time="0.009" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Create_Type" executed="True" success="True" time="0.010" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Delete_deletes_objects" executed="True" success="False" time="0.212" asserts="2">
                                <failure>
                                  <message><![CDATA[System.InvalidCastException : Das Objekt des Typs "System.Int32" kann nicht in Typ "Kistl.API.IDataObject" umgewandelt werden.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.API.Server.QueryTranslatorProvider`1.Execute(Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 114.
bei Kistl.API.Server.QueryTranslatorProvider`1.Execute[TResult](Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 104.
bei System.Linq.Queryable.Count[TSource](IQueryable`1 source)
bei Kistl.DalProvider.EF.Tests.KistlDataContextTests.Delete_deletes_objects() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\KistlDataContextTests.cs:Zeile 526.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Delete_triggers_ObjectDeleted" executed="True" success="True" time="0.029" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Detach_IDataObject" executed="True" success="True" time="0.026" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Detach_IDataObject_Failed" executed="True" success="True" time="0.009" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_ObjectType_fails_on_invalid_ID" executed="True" success="False" time="0.143" asserts="0">
                                <failure>
                                  <message><![CDATA[System.InvalidCastException : Das Objekt des Typs "System.Int32" kann nicht in Typ "Kistl.API.IDataObject" umgewandelt werden.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.DALProvider.EF.KistlDataContext.Find(Type type, Int32 ID) in P:\Kistl\Kistl.DALProvider.EF\KistlDataContext.cs:Zeile 338.
bei Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_ObjectType_fails_on_invalid_ID() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\KistlDataContextTests.cs:Zeile 171.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_ObjectType_returns_right_object" executed="True" success="True" time="0.015" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_T_fails_on_invalid_ID" executed="True" success="False" time="0.025" asserts="0">
                                <failure>
                                  <message><![CDATA[An unexpected exception type was thrown
Expected: System.ArgumentOutOfRangeException
 but was: System.InvalidCastException : Das Objekt des Typs "System.Int32" kann nicht in Typ "Kistl.API.IDataObject" umgewandelt werden.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.API.Server.QueryTranslatorProvider`1.Execute(Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 114.
bei Kistl.API.Server.QueryTranslatorProvider`1.Execute[TResult](Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 104.
bei System.Linq.Queryable.Count[TSource](IQueryable`1 source, Expression`1 predicate)
bei Kistl.DALProvider.EF.KistlDataContext.Find[T](Int32 ID) in P:\Kistl\Kistl.DALProvider.EF\KistlDataContext.cs:Zeile 368.
bei Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_T_fails_on_invalid_ID() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\KistlDataContextTests.cs:Zeile 150.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.Find_T_returns_right_object" executed="True" success="True" time="0.017" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetContext_returns_a_context" executed="True" success="True" time="0.009" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_ObjType_returns_a_non_empty_list_on_class_DataType" executed="True" success="True" time="0.711" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_ObjType_WrongPropertyName_fails" executed="True" success="True" time="0.024" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_ObjType_WrongType_fails" executed="True" success="True" time="0.049" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_T_SubClasses_returns_a_non_empty_list_on_class_DataType" executed="True" success="True" time="0.040" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_T_WrongPropertyName_fails" executed="True" success="True" time="0.019" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetListOf_T_WrongType_fails" executed="True" success="True" time="0.038" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetQuery" executed="True" success="True" time="0.009" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.GetQuery_ObjType" executed="True" success="True" time="0.052" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.InitSession_returns_a_context_and_inits_Current" executed="True" success="True" time="0.009" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.InitSessionTwice_creates_a_new_context" executed="True" success="True" time="0.009" asserts="2" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.NoSession" executed="True" success="True" time="0.008" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.KistlDataContextTests.UpdateSomeData_SubmitChanges" executed="True" success="True" time="0.029" asserts="4" />
                            </results>
                          </test-suite>
                          <test-suite name="Linq" success="False" time="0.075" asserts="0">
                            <results>
                              <test-case name="Kistl.DalProvider.EF.Tests.Linq.Count_returns_correct_count" executed="True" success="False" time="0.018" asserts="0">
                                <failure>
                                  <message><![CDATA[System.InvalidCastException : Das Objekt des Typs "System.Int32" kann nicht in Typ "Kistl.API.IDataObject" umgewandelt werden.]]></message>
                                  <stack-trace><![CDATA[bei Kistl.API.Server.QueryTranslatorProvider`1.Execute(Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 114.
bei Kistl.API.Server.QueryTranslatorProvider`1.Execute[TResult](Expression expression) in P:\Kistl\Kistl.API.Server\QueryTranslator.cs:Zeile 104.
bei System.Linq.Queryable.Count[TSource](IQueryable`1 source)
bei Kistl.DalProvider.EF.Tests.Linq.<Count_returns_correct_count>b__2(IQueryable`1 q) in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\Linq.cs:Zeile 85.
bei Kistl.DalProvider.EF.Tests.Linq.Count_returns_correct_count() in P:\Kistl\Tests\Kistl.DalProvider.EF.Tests\Tests\Linq.cs:Zeile 88.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.Linq.Where_filters_correctly" executed="True" success="True" time="0.051" asserts="2" />
                            </results>
                          </test-suite>
                          <test-suite name="ServerCollectionEntry_EF" success="False" time="0.027" asserts="0">
                            <results>
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.fails_on_serializing_from_null_stream" executed="True" success="True" time="0.002" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.fails_on_serializing_to_null_stream" executed="True" success="True" time="0.000" asserts="0" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_be_attached_after_attaching" executed="True" success="False" time="0.000" asserts="0">
                                <failure>
                                  <message><![CDATA[Cannot get context without Kistl.API.{Client,Server} reference]]></message>
                                  <stack-trace><![CDATA[bei Kistl.API.Tests.Skeletons.IPersistenceObjectTests`1.should_be_attached_after_attaching() in P:\Kistl\Tests\Kistl.API.Tests\Tests.Skeletons\IPersistenceObjectTests.cs:Zeile 27.
]]></stack-trace>
                                </failure>
                              </test-case>
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_be_attached_to_EFContext_after_attaching" executed="True" success="True" time="0.005" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_have_a_RelationshipManager" executed="True" success="True" time="0.001" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_have_Detached_EntityState_when_created" executed="True" success="True" time="0.001" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_have_no_EntityKey_when_created" executed="True" success="True" time="0.000" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_not_be_attached_when_created" executed="True" success="True" time="0.000" asserts="1" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_roundtrip_IPersistenceObject_correctly" executed="True" success="True" time="0.006" asserts="3" />
                              <test-case name="Kistl.DalProvider.EF.Tests.ServerCollectionEntry_EF.should_roundtrip_without_errors" executed="True" success="True" time="0.005" asserts="1" />
                            </results>
                          </test-suite>
                        </results>
                      </test-suite>
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>